eng/scripts/Language-Settings.ps1

```diff
@@ -201,6 +201,33 @@ $PackageExclusions = @{
   'azure-monitor-query' = 'Unsupported doc directives https://github.com/Azure/azure-sdk-for-python/issues/19417';
   'azure-mgmt-network' = 'Manual process used to build';
 }
+
+# In python, we download the dev version from custom feed in public/azure-sdk-for-python
+function Get-python-DocsMsDevLanguageSpecificPackageInfo($packageInfo) {
+  try
+  {
+    $pythonPackageInfo = Invoke-RestMethod -Uri "https://dev.azure.com/azure-sdk/public/_packaging?_a=package&feed=azure-sdk-for-python&package=$($packageInfo.Name)&protocolType=PyPI&version=$($packageInfo.DevVersion)#"
+
+    if ($pythonPackageInfo.'dist-tags'.dev)
+    {
+      Write-Host "Using published version at 'dev' tag: '$($npmPackageInfo.'dist-tags'.dev)'"
```

Comment:
This looks like it's implemented for npm... We may not need to implement this at all if we publish the Python package every time (this was done for JS so we could find the actual published JS package version which might be different from the dev version generated for this build).

Reply:
Right, I have commented out this, but merged back when I do clean up. I will make it specific for python

Reply:
I removed this completely. :)